pytorch_lightning/accelerators/cpu.py

```diff
@@ -48,3 +53,27 @@ def auto_device_count() -> int:
     def is_available() -> bool:
         """CPU is always available for execution."""
         return True
+
+
+class CPUDeviceStatsEnum(LightningEnum):
+    """Enum for CPU device stats."""
+
+    CPU_VM_PERCENT = "cpu_vm_percent"
+    CPU_PERCENT = "cpu_percent"
+    CPU_SWAP_PERCENT = "cpu_swap_percent"
+
+
+def get_cpu_process_metrics() -> dict[str, float]:
```

Comment:
```suggestion
def get_cpu_process_metrics() -> Dict[str, float]:
```

Reply:
Pre-commit bot auto changes `Dict` to `dict`: https://pytorch-lightning.slack.com/archives/C01296PBDQV/p1644417099319789